src/app/screens/SignUp/index.tsx

```diff
@@ -2,98 +2,91 @@ import React from 'react';
 import i18next from 'i18next';
 import { useForm } from 'react-hook-form';
 
+import { SIGN_UP_FIELDS } from './constants';
 import styles from './styles.module.scss';
 import WoloxImg from './assets/wolox-logo.png';
+import CustomInput from './components/CustomInput';
 
-interface User {
-  first_name: string;
-  last_name: string;
+interface UserProps {
```

Comment:
this is not "Props", is more like the fields or the payload. Be careful about naming